source/_docs/scripts.markdown

```diff
@@ -657,7 +657,10 @@ automation:
 
 {% endraw %}
 
-`conditions` also accepts a [shorthand notation of a template condition][shorthand-template].
+`conditions` also accepts a [shorthand notation of a template condition][shorthand-template]. <br>
+It needs to be wrapped in quotation marks (```""```) unless using folding (```>```) where it must not use quotation marks. When wrapping in outside double quotation marks ensure any inner quotation marks are of the single (```'```) variant. <br>
+Note: use ```conditions``` in contrast with ```condition``` when using [template shorthand notation](https://www.home-assistant.io/docs/scripts/conditions/#template-condition-shorthand-notation).
```

Comment:
- This is incorrect and not full.
- Don't use bare links.

Reply:
- I'm just trying to update the docs with more info if I encounter struggles, to aid new users. I'm by no means proficient let alone an expert. What is incorrect?
- OK, new to me. I personally like it when it's clear from the link description but happy to change it.